packages/wonder-blocks-modal/src/components/modal-dialog.tsx

```diff
@@ -36,6 +36,11 @@ type Props = {
      * Test ID used for e2e testing.
      */
     testId?: string;
+    /**
+     * The accessible name of dialog.
+     * See WCAG 2.1: 4.1.2 Name, Role, Value
+     */
+    "aria-label"?: string;
```

Comment:
Note: `aria-label` is supported in dialogs, so I added it here since `FlexibleDialog` builds on top. I didn't mess with `aria-labelledby` being required to limit the scope here, but it really should require one or the other.